tests/gen_pickles.py

```diff
@@ -19,8 +19,9 @@ def write(name, proto):
 
 
 def generate():
-    if not USE_CYTHON:
-        raise RuntimeError("Cython is required")
+    if not USE_C_EXTENSIONS:
+        # TODO: mb this useless and should be refactored
+        raise RuntimeError("C is required")
```

Comment:
C? Does a normal user need to know what that is? GCC I guess

Reply:
Good point. I will change it.